R/utils.R

```diff
@@ -4,6 +4,14 @@ line_col_names <- function() {
   c("line1", "line2", "col1", "col2")
 }
 
+styler_df <- function(..., .size = NULL, .name_repair = "minimal") {
```

Comment:
I am not sure we should use these helpers. They seem to be (too) thin wrappers around {vtrcs} functions without deviating defaults. I'd suggest to just use those directly. What is the rationale to wrap?

Reply:
It provides a single encapsulation point that we can replace with tibble once it catches up with base-R in terms of performance.

Reply:
This was suggested by Kirlll [here](https://github.com/r-lib/styler/pull/1007#pullrequestreview-1120582454).

Reply:
Ok, if he thinks so I won't oppose :D